src/pages/waiting-room-to-car/waiting-room-to-car.ts

```diff
@@ -276,9 +276,10 @@ export class WaitingRoomToCarPage extends PracticeableBasePageComponent {
     if (this.form.valid) {
       this.navController.push(TEST_REPORT_PAGE).then(() => {
         // remove Waiting Room To Car Page
-        this.navController.removeView(
-          this.navController.getViews().find(view => view.name === WAITING_ROOM_TO_CAR_PAGE),
-        );
+        const view = this.navController.getViews().find(view => view.id === WAITING_ROOM_TO_CAR_PAGE);
+        if (view) {
```

Comment:
How come we have to check if it exists? Should we handle the case where it doesn't?

Reply:
In this scenario the page _should_ exist as we're navigating off it, but the conditional handles the case when we can't find the view in the nav stack e.g. because the `id` or `name` property is no longer available or minified.